bigchaindb/consensus.py

```diff
@@ -89,6 +89,18 @@ def verify_signature(signed_transaction):
         """
         raise NotImplementedError
 
+    @abstractmethod
```

Comment:
put in refactor: of course!

any reason why you would disagree on this?

Not specifying the `@staticmethod` decorator might trick child classes that the first argument is a `self` but that's not the case